cloudflare/resource_cloudflare_record_migrate_test.go

```diff
@@ -147,7 +174,7 @@ func TestCloudflareRecordMigrateState(t *testing.T) {
 		}
 
 		if is.ID != tc.Expected {
-			t.Fatalf("bad sg rule id: %s\n\n expected: %s", is.ID, tc.Expected)
+			t.Fatalf("bad record id: %s\n\n expected: %s", is.ID, tc.Expected)
```

Comment:
This came from https://github.com/terraform-providers/terraform-provider-aws/blob/v2.43.0/aws/resource_aws_security_group_rule_migrate_test.go#L64

Reply:
great pickup! thanks!